src/checkpoint/checkpoint.h

```diff
@@ -50,7 +48,25 @@ struct __wt_ckpt_connection {
     uint64_t meta_check_time; /* Checkpoint handles metadata check time */
     uint64_t skip;            /* Checkpoint handles skipped */
     uint64_t skip_time;       /* Checkpoint skipped handles gather time */
-    uint64_t usecs;           /* Checkpoint timer */
+};
+
+/*
+ * WT_CKPT_CONNECTION --
+ *     Checkpoint information.
+ */
+struct __wt_ckpt_connection {
+    WT_SESSION_IMPL *session;       /* Checkpoint thread session */
+    wt_thread_t tid;                /* Checkpoint thread */
+    bool tid_set;                   /* Checkpoint thread set */
+    WT_CONDVAR *cond;               /* Checkpoint wait mutex */
+    wt_shared uint64_t most_recent; /* Clock value of most recent checkpoint */
+#define WT_CKPT_LOGSIZE(conn) (__wt_atomic_loadi64(&(conn)->ckpt.logsize) != 0)
+    wt_shared wt_off_t logsize; /* Checkpoint log size period */
+    bool signalled;             /* Checkpoint signalled */
+
+    WT_CKPT_HANDLE handle;
```

Comment:
Should I make this plural?

```suggestion
    WT_CKPT_HANDLE handles; /* Checkpoint handles */
```

Reply:
The content of the struct doesn't look like there are many handles.
And it doesn't seem to be a handle. I'd rather call it "stats" or something like that.

Reply:
I am not sure if I understand correctly your comment or I misunderstood the fields. AFAIK, those fields reflect stats (indeed) about all the dhandles the checkpoint is currently dealing with. It could be one or thousands.

Reply:
Sorry, after seeing @sueloverso's [comment](https://github.com/wiredtiger/wiredtiger/pull/11347/files#r1890500930) and looking at other structures we have in the code, your comment makes sense @ershov, sorry.
@sueloverso, it seems that we use *_info much less than *_stats, would you be ok with `WT_CKPT_HANDLE_STATS`?

Reply:
I'm good with _STATS

Reply:
Renamed in 6f129f909c266f0753a5750da177c291531e54fd